de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/ui/wizards/AddProjectToSessionWizard.java

```diff
@@ -86,7 +87,7 @@
     private final String remoteProjectID;
     private final String remoteProjectName;
 
-    private final AbstractIncomingProjectNegotiation negotiation;
+    private final AbstractIncomingReferencePointNegotiation negotiation;
     private final JID peer;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: "peer" is the name of a field in "Component". ([squid:S2387](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS2387))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWcmJq3SkEeD8QRkTSun&open=AWcmJq3SkEeD8QRkTSun)

[ISSUE_KEY]: AWcmJq3SkEeD8QRkTSun